src/services/mentees.js

```diff
@@ -958,6 +958,10 @@ module.exports = class MenteesHelper {
 				}
 			}
 
+			if (organization.toLowerCase() === common.FALSE) {
```

Comment:
@sumanvpacewisdom first why ur fetching the org details if its not require in case of organisation is set false